build/dependencies.props

```diff
@@ -1,21 +0,0 @@
-<Project>
-  <PropertyGroup>
-    <AspNetCoreIntegrationTestingVersion>0.5.0-*</AspNetCoreIntegrationTestingVersion>
-    <AspNetCoreVersion>2.1.0-*</AspNetCoreVersion>
-    <CoreFxLabsVersion>0.1.0-*</CoreFxLabsVersion>
-    <CoreFxVersion>4.4.0-*</CoreFxVersion>
-    <GoogleProtobufVersion>3.1.0</GoogleProtobufVersion>
-    <InternalAspNetCoreSdkVersion>2.1.1-*</InternalAspNetCoreSdkVersion>
-    <JsonNetVersion>10.0.1</JsonNetVersion>
-    <MoqVersion>4.7.49</MoqVersion>
-    <NETStandardImplicitPackageVersion>2.0.0-*</NETStandardImplicitPackageVersion>
-    <NETStandardLibraryNETFrameworkVersion>2.0.0-*</NETStandardLibraryNETFrameworkVersion>
-    <RedisVersion>1.2.4</RedisVersion>
-    <RuntimeFrameworkVersion Condition="'$(TargetFramework)'=='netcoreapp2.0'">2.0.0-*</RuntimeFrameworkVersion>
-    <TestSdkVersion>15.3.0</TestSdkVersion>
-    <XunitVersion>2.3.0-beta4-build3742</XunitVersion>
-    <RxVersion>3.1.1</RxVersion>
-    <MsgPackVersion>0.9.0-beta2</MsgPackVersion>
```

Comment:
Where do these version come from now? Especially SignalR specific like MsgPack or Rx? How do I update the version or add a new dependency?

Reply:
Because SignalR is building in "Universe", these packages are coming from the Universe lineup.

https://github.com/aspnet/Universe/blob/590d42dc88911fe5d9d9894c4e9c881161d6af0c/build/dependencies.props#L70